apps/design-app/src/app/app.module.ts

```diff
@@ -0,0 +1,18 @@
+import { BrowserModule } from '@angular/platform-browser';
+import { NgModule } from '@angular/core';
+
+import { AppComponent } from './app.component';
+import { DaffLoadingIconModule } from 'libs/design/src';
```

Comment:
This should reference `@daffodil/design`, but this branch also doesn't seem to be branched off of the "Move loading-icon into design module". So this throws an error, because loading-icon is still in `demo`. Should be ok if this PR just waits on the loading-icon PR

Reply:
@lderrickable this is ready for re-review.